Makefile

```
@@ -1,4 +1,19 @@
-.PHONY: dev deviex signoff signoff-dirty signoff-stack merge help init spellcheck addword check-links deploy buildpush buildpush-dirty remiex connectdb connect docs redis-console-consumer
+# Running `make` will trigger `make help`
+.DEFAULT_GOAL := help
+
+# for formatting
+BOLD="\033[1m"
+NORMAL="\033[0m"
+
+# Any target invoked by make target does not correspond to a file, so
+# it should always be executed. This avoids: 1. Having to remember to
+# add newly created targets as PHONY.  2. Using a catch-all target
+# (%:) with a no-op (@:) to prevent failures for non-existing targets.
+.PHONY: $(MAKECMDGOALS)
+
+help: ## Prints target: [dep1 dep1 ...]  and what it does
+	@echo -e ${BOLD}
```

@oneness Is this line incomplete? Perhaps you were trying to title the help output, like this?

```suggestion
	@echo ${BOLD}Available targets${NORMAL}
```

---

Good catch!  Seems like you are reading my mind :).  
FYI: pushed the fix